### PR TITLE
Multiple Argo support using the app-selector annotation

### PR DESCRIPTION
**Pull request by sam-robson**
Following [this issue](https://github.com/RoadieHQ/roadie-backstage-plugins/issues/289), this should allow users to scan multiple Argos using the app-selector annotation; allowing them to display multiple environments across multiple instances in the plugin.

#### :heavy_check_mark: Checklist

- [X] Added tests for new functionality and regression tests for bug fixes
- [ ] Screenshots of before and after attached (for UI changes)
- [ ] Added or updated documentation (if applicable)

### CODE REVIEW

File: plugins/backend/backstage-plugin-argo-cd-backend/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@roadiehq/backstage-plugin-argo-cd-backend",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (Xantier) [inline plugins/backend/backstage-plugin-argo-cd-backend/package.json:3]: Let's bump this and the frontend plugin with a minor version change instead.

**Review by Xantier — COMMENTED** [review]
Could you maybe add few lines into the respective README.md files about this change as well?

**Comment by Xantier** [comment]
@sam-robson, friendly nudge: did you get a chance to take another look at this?

**Comment by sam-robson** [comment]
> @sam-robson, friendly nudge: did you get a chance to take another look at this?

Sorry I was on holiday. I'll do this asap.

**Comment by sam-robson** [comment]
> Could you maybe add few lines into the respective README.md files about this change as well?

On further reflection, I'm not sure if any changes to the README are necessary. The current README doesn't specify whether the app-name or app-selector annotation is required, and all I've done is allow users to use either. Does this make sense?

**Comment by dtuite** [comment]
Hi @sam-robson. Xantier (Jussi) has been out for a few days so we are a little slow getting this merged. We'll take a look soon!

**Comment by Xantier** [comment]
Hi @sam-robson, 
apologies for the back and forth on this. There are few more things on this, Instead of bumping the versions directly in package.json files, could you add a `minor` changeset for both BE and FE ArgoCD plugins. With that in we'll get automatic release notes. I tried to merge the latest master into this branch already but there looks to be one more conflict that have crept in. 

With those in, we'll be able to get this shipped.

Thanks!

**Review by Xantier — APPROVED** [review]
Excellent! Thanks @sam-robson .
Merging..

**Comment by sam-robson** [comment]
Great! Thanks for the assistance.

**Comment by yanyuxuanz** [comment]
how to use it (app-selector) in a entity,.

**Pull request merged**